pnpm-lock.yaml

```diff
@@ -13762,14 +13809,14 @@ packages:
       '@typescript-eslint/parser':
         optional: true
     dependencies:
-      '@typescript-eslint/parser': 6.2.1(eslint@8.40.0)(typescript@5.1.3)
+      '@typescript-eslint/parser': 5.59.2(eslint@8.40.0)(typescript@5.1.3)
```

Comment:
why is this getting downgraded lol

Reply:
There are other changes in the file that where needed for the new example "examples/minimal-custom-server" to pass all tests. I can only assume the file has been updated several times since I made the amends